src/main/java/rx/observers/TestSubscriber.java

```diff
@@ -356,10 +356,12 @@ public void assertError(Class<? extends Throwable> clazz) {
             throw new AssertionError("No errors");
         } else
         if (err.size() > 1) {
-            throw new AssertionError("Multiple errors: " + err.size(), new CompositeException(err));
+            // can't use AssertionError because (message, cause) doesn't exist until Java 7
```

Comment:
Cool. Updated. I have not updated `assertNoErrors` because changing from RuntimeException to AssertionError is a breaking change.